<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OOP Lab1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>600</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_Move_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>300</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Масштабирование</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_ScaleX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>kx</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ScaleY">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>ky</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ScaleZ">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>kz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Scale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Масштабировать</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_Move_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>340</y>
      <width>300</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string>Поворот</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_TurnX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>ox</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_TurnY">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>oy</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_TurnZ">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>oz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Turn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Повернуть</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>cx</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CY">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>cy</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CZ">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>cz</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Загрузить файл</string>
    </property>
    <widget class="QLineEdit" name="Filename">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>имя файла</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UploadButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Загрузить</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_Move">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>300</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Перенос</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_MoveX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>dx</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_MoveY">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>dy</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_MoveZ">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>dz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Move">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Перенести</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>964</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_7_42">
    <property name="title">
     <string>Чалый А.А. ИУ7-42</string>
    </property>
   </widget>
   <addaction name="menu_7_42"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
